R/suffix.R

```diff
@@ -142,8 +142,9 @@ suffix_extract <- function(domains, suffixes = NULL){
         stop("Expected column named \"suffixes\" in suffixes data.frame")
       }
     }
-    suffix_load(suffixes)
   }
+  suffix_load(suffixes)
```

Comment:
The other option is:
- Keep this `suffix_load(suffixes)` inside the `if(!is.null(suffixes)`
- Add to the end of the function a cleanup routine:
````
if(!is.null(suffixes) {
    suffix_load(NULL)
}````

I think reloading every time the function is run is "cleaner", but I'll agree it is not the most efficient if someone never uses the custom suffixes.